fresque/forms.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
+    # -*- coding: utf-8 -*-
```

Comment:
Sorry, I missed that. Now onwards, I will be more careful before a committing these changes.